README.md

```diff
@@ -16,6 +16,8 @@ With the Azure login Action, you can automate your workflow to do an Azure login
 
 By default, the action only logs in with the Azure CLI (using the `az login` command). To log in with the Az PowerShell module, set `enable-AzPSSession` to true. To login to Azure tenants without any subscriptions, set the optional parameter `allow-no-subscriptions` to true.
 
+Incase user wants to switch the context to Azure US Government Cloud or Azure China Cloud then azlogin action supports it from version 1.2.This can be achieved by passing an extra environment variable with the name of the cloud which user wants to connect .The action azlogin expects 'AzureUSGovernment' or 'AzureChinaCloud' depending on the cloud which user is trying to connect. By default it connects to Azure Public Cloud which can also be specified as 'AzureCloud' in environment variable.It also needs a  service principal created in the particular cloud to connect (Please refer Configure deployment credentials section for details) 
```

Comment:
To login into one of the [Azure Government clouds](https://docs.microsoft.com/en-us/azure/azure-government/documentation-government-welcome), set the optional parameter `environment` with supported cloud names `AzureUSGovernment` or `AzureChinaCloud`. If this parameter is not specified, it takes the default value `AzureCloud`  and connect to the Azure Public Cloud. Additionally the parameter `creds` takes the [Azure service principal](https://docs.microsoft.com/azure/active-directory/develop/app-objects-and-service-principals) created in the particular cloud to connect (Refer to **Configure deployment credentials** section below for details).